.nowignore

```diff
@@ -0,0 +1,9 @@
+*
```

Comment:
ignore everything then unignore the files we want as per the docs: https://zeit.co/docs/v2/platform/upgrade-to-2-0/#.nowignore